Gemfile

```diff
@@ -9,7 +9,7 @@ end
 
 group :test, :development do
   gem 'rake', '>= 0.7.3'
-  gem 'rspec', '>= 2.9.0'
+  gem 'rspec', '~> 2.14.0'
```

Comment:
I changed this `rspec` version since the current Gemfile was causing it to use the latest rspec release. Unfortunately, that caused almost all of the tests to fail, as there are a number of deprecated rspec uses in the current tests. I briefly tried updating the tests, but decided this was a better short-term solution for this PR. Version 2.14 seems to be the latest in the 2.x line.